running/ambassador-edge/routing/bring-recycling/prod.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: bring-recycling-node-mapping
+  namespace: ambassador-edge
+spec:
+  host: bringrecycling.mvpstudio.org
+  prefix: /api
+  service: bring-recycling-webserver.prod-bring-recycling
+---
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: bring-recycling-react-mapping
+  namespace: ambassador-edge
+spec:
+  host: bringrecycling.mvpstudio.org
+  prefix: /
+  service: bring-recycling-react.prod-bring-recycling
+---
+apiVersion: getambassador.io/v2
+kind: Host
+metadata:
+  name: bring-recycling-host
+  namespace: ambassador-edge
+spec:
+  hostname: bringrecycling.mvpstudio.org
+  acmeProvider:
+    email: mvpstudiooregon+bringrecycling@gmail.com
```

Comment:
Note I changed this line per https://eugenetech.slack.com/archives/C9J4G6F5Y/p1607303121297200

Will change other emails and update docs in another PR.